README.md

```diff
@@ -404,6 +410,21 @@ CMS are a practical way to setup a website with many features. CMS often come wi
   * [MaterialPoll](https://github.com/nearbycoder/materialpoll) - an opensource strawpoll.me clone for creating random polls. - ([Source code](https://github.com/nearbycoder/materialpoll), [Demo](http://materialpoll.tk))
   * [OpenSondage](https://github.com/leblanc-simon/OpenSondage) - an online service for planning an appointment or make a decision quickly and easily: Make a poll, Define dates or subjects to choose, Send the poll link to your friends or colleagues, Discuss and make a decision - ([Demo](http://framadate.org/), [Example](https://framadate.org/aqg259dth55iuhwm))
 
+## Project Management
```

Comment:
You are right, but then we would have to sort between dev and non-dev oriented software...

What can be done: move this to Software Development/Project Management subsection but leave the section header here, and just link to the other one:

``See [Software Development Platforms/Project Management](#project-management)``